priv/repo/migrations/20220112130310_create_arenas.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Sports.Repo.Migrations.CreateArenas do
+  use Ecto.Migration
+
+  def change do
+    create table(:arenas) do
+      add :opens_at, :integer
+      add :closes_at, :integer
+      add :name, :string
```

Comment:
Add "not null" clauses to the fields which are required for additional database validations